src/app/components/Social/index.tsx

```diff
@@ -11,17 +11,23 @@ import twitter from './images/twitter.svg'
 import discord from './images/discord.svg'
 import youtube from './images/youtube.svg'
 import reddit from './images/reddit.svg'
+import LinkedInIcon from '@mui/icons-material/LinkedIn'
+import DocsIcon from '@mui/icons-material/MenuBook'
+import HomeIcon from '@mui/icons-material/Cottage'
 import { COLORS } from '../../../styles/theme/colors'
 import { socialMedia } from '../../utils/externalLinks'
+import { OverridableComponent } from '@mui/material/OverridableComponent'
+import { SvgIconTypeMap } from '@mui/material/SvgIcon'
 
 type SocialLinkProps = {
   label: string
   href: string
   isMobile: boolean
-  img: string
+  imgSrc?: string
+  img?: OverridableComponent<SvgIconTypeMap> & { muiName: string }
```

Comment:
Can we just pass an icon component as ReactNode and also remove React.createElement ?

Reply:
Done that now.